### PR TITLE
Fixed footer bug

### PR DESCRIPTION
**Pull request by pkakelas**
Βug...
screenshot from 2014-03-19 14 57 51
Fixed.
screenshot from 2014-03-19 14 59 48

### CODE REVIEW

**Comment by dionyziz** [comment]
Please upload screenshots for before and after the bugfix.

File: static/style/general.css [hunk 1]
```diff
@@ -9,3 +9,6 @@
 #avatar-form {
     margin-left: 46%;
 }
+#main {
+    margin-bottom: 5%;
```
Comment (dionyziz) [inline static/style/general.css]: I think you want to use an absolute value for this. Isn't the footer static in height regardless of window size? This means you need to make two changes - one here for the margin, and one for the actual size of the footer height.

File: static/style/general.css [hunk 2]
```diff
@@ -9,3 +9,6 @@
 #avatar-form {
     margin-left: 46%;
 }
+#main {
+    margin-bottom: 65px;
+}
```
Comment (dionyziz) [inline static/style/general.css]: Move to same file that determines footer height to ensure orthogonality between files.

**Comment by dionyziz** [comment]
I don't think `position: static` will work well with pages of short height. Can you post a screenshot with a page that doesn't fill the screen vertically?

File: static/style/general.css (hunk 2 above)
Comment (dionyziz) [inline static/style/general.css]: I reiterate this comment.

Comment (pkakelas) [inline static/style/general.css]: I don't believe that it is appropriate to move this code to footer.css
becsuse this property is about the main div of the page and not the footer
Óôéò 22 Ìáñ 2014 1:59 ð.ì., ï ÷ñÞóôçò "Dionysis Zindros" <
notifications@github.com> Ýãñáøå:

Comment (dionyziz) [inline static/style/general.css]: The property of `margin-bottom` concerns the footer of the page. `main` is making space for the footer. If the footer changes in height, main's margin should also change. We should keep these changes in the same file.

**Pull request merged**